WcaOnRails/app/webpacker/components/Media/MediaTable.jsx

```diff
@@ -0,0 +1,119 @@
+import _ from 'lodash'
+import React, { useState, useEffect } from 'react';
+import { Table } from "semantic-ui-react";
+import { countries } from '../../lib/wca-data.js.erb';
+import { competitionUrl } from '../../lib/requests/routes.js.erb';
+import moment from 'moment';
+
+function dateRangeBetween(fromDate, toDate) {
+    const format = new Intl.DateTimeFormat("en", {
+        year: "numeric",
+        month: "short",
+        day: "numeric",
+    });
+    const date1 = new Date(fromDate);
+    const date2 = new Date(toDate);
+    const ans = format.formatRange(date1, date2);
+    return ans;
+}
+
+function exampleReducer(state, action) {
```

Comment:
It would be good to have proper naming of variables in this situation.

Reply:
Fixed